### ISSUE TITLE
Mac OS 몬터레이에서 scouter 실행 불가

### ISSUE THREAD

**Issue by rhdtl78** [scouter-project/scouter#898]
![스크린샷 2022-10-19 10 54 02](https://user-images.githubusercontent.com/28679823/196578823-9a0791a6-86fc-4bb1-8b04-28e441cbaf86.png)

> The JVM shared library "/Library/Java/JavaVirtualMachines/temurin-11.jdk/Contents/Home/bin/../lib/server/libjvm.dylib"
does not contain the JNI_CreateJavaVM symbol.

xattr -cr scouter.client.app 이후, 더블 클릭으로 실행했습니다.

scouter 에서 찾지 못한다고 하는 파일의 경로를 따라가보니 파일이 있습니다.

뭔가 놓친게 있을까요?

> Environments
> scouter version : 2.17.1
> mac os version : macOS Monterey 12.6
> Architecture : arm64 (M1 Pro)
> JDK Version : termurin JDK 11.0.15

**Comment by sejoung** [comment]
저도 똑같은 상황이네요 혹시 해결된 내용 있을까요?

**Comment by zbum** [comment]
@rhdtl78 @sejoung 
arm64 바이너리로 클라이언트 빌드 성공했습니다. 
곧 PR 올릴께요~

**Comment by sejoung** [comment]
@zbum 감사합니다. 👍 🙇 ❤️

**Comment by zbum** [comment]
#907

**Comment by zbum** [comment]
@sejoung  관리자님이 바쁘신것 같아 제 fork 에 SNAPSHOT 릴리즈를 올려 두었습니다. 필요하시면 활용하세요.

https://github.com/zbum/scouter/releases/tag/2.18.0-SNAPSHOT

**Comment by sejoung** [comment]
@zbum 네 처음에 PR 올리시고 몇일뒤에 코드로 빌드 하려고 찾아갔는데 있어서 잘쓰고 있습니다. 감사합니다. 🙇  좋은 하루보내세요 👍

**Comment by childeye** [comment]
<img width="252" alt="image" src="https://github.com/scouter-project/scouter/assets/42136685/5a56b853-4743-461b-a03b-6f88a3a45bac">

# Environments
 - scouter version : v2.20.0
 - mac os version : macOS Monterey 12.6.1
 - Architecture : arm64 (M1 Pro)

$ java -version
openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)

최신 버전 다운로드 받아 설치했는데 에러 발생합니다. ㅠ
